src/commands/analyze-lambda-cold-starts.js

```diff
@@ -56,7 +56,7 @@ class AnalyzeLambdaColdStartsCommand extends Command {
 	}
 
 	async getFunctionsInRegion(region) {
-		const functionDetails = await Lambda.getFunctionsInRegion(region);
+		const functionDetails = await Lambda.getFunctionsInRegion(region, getAWSSDK());
```

Comment:
why did you make this change?

Reply:
I'm using `aws-sdk-mock` see https://github.com/dwyl/aws-sdk-mock and it's a required change for a successful mock. Do you see a problem with the change? It's classical IoC, no?

Reply:
I've been using jest to mock AWS SDK interactions up to now, would be good if you could use the same approach so the project is internally consistent. Unless there's somethings you can only do with aws-sdk-mock?